### DISCUSSION TITLE
WinUI OnActivated doesn't exist

### DISCUSSION THREAD

**Discussion by Soap-141** [unoplatform/uno#10803] [Q&A] [answered]
Hello,

I was wondering where is the entry point when launching iOS and Android from URLS and push notifications in the new WinUI .NET 6 template?

In UWP it was the `OnActivated` event but it doesn't exist anymore in WinUI, is it inside the `OnLaunched` event?

https://github.com/nventive/UnoApplicationTemplate/pull/251#discussion_r1054633103

**Comment by Soap-141** [comment, marked as answer]
Found this but Uno instruction would be great. https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/migrate-to-windows-app-sdk/guides/toast-notifications?tabs=appsdk

**Reply by jeromelaban** [reply]
OnActivated does not exist on WinUI, but still does exist on Uno, you can use `#if HAS_UNO` to access it. We haven't added the WinAppSDK APIs yet, that will come soon.

**Reply by Soap-141** [reply]
Yes as discussed latest Uno TouchPoint. Thanks!
